data/SkyPy/lsst-like_new.yml

```diff
@@ -0,0 +1,107 @@
+# These are the luminosity function parameters according to M = a_m * z + b_m
+# and Phi = b_phi * exp(a_phi * z). As in Lopez-Sanjuan the model is given as
+# M = M0 + Q * (z - 0.5) and log10(Phi) = Phi0 + P * (z - 0.5) we have to convert
+# the given parameters as a_m = Q, b_m = M0 - 0.5*Q, a_phi = ln(10) * P and
+# b_phi = 10**(Phi0 - 0.5*P)
+cosmology: !astropy.cosmology.default_cosmology.get []
+z_range: !numpy.arange [0.0, 5.01, 0.04]
+M_star: !astropy.modeling.models.Linear1D [-1.03, -20.485]
```

Comment:
are these M_star for the blue or red distribution? Perhaps specify these parameters better